test/std-test.cc

```diff
@@ -90,6 +90,34 @@ TEST(std_test, optional) {
 #endif
 }
 
+namespace my_nso {
```

Comment:
What does "nso" stand for?

Reply:
> What does "nso" stand for?

I was trying to follow the naming scheme with short namespaces below, `my_ns1` & `my_ns2`, and added an `o` for `optional`, since `my_ns3` before the two others didn't make sense. But I'll change to whatever you prefer.

Reply:
`my_nso` is OK